regex-capi/Cargo.toml

```diff
@@ -11,7 +11,7 @@ description = """
 A C API for Rust's regular expression library.
 """
 workspace = ".."
-edition = "2018"
+edition = "2021"
```

Comment:
Probably set `rust-version = "1.65"` for consistency with other crates?

Reply:
done